doc/man1/openssl-kdf.pod.in

```diff
@@ -66,7 +66,7 @@ cases.
 =item B<-kdfopt> I<nm>:I<v>
 
 Passes options to the KDF algorithm.
-A comprehensive list of parameters can be found in the EVP_KDF_CTX
```

Comment:
can be found in L<,,,>.
The implementaion documentation bit is not needed.

Reply:
done